lib/api/dialogs.js

```diff
@@ -0,0 +1,14 @@
+export default function createDialogs (channel) {
+  return {
+    selectSingleEntry: call.bind(null, 'Entry', false),
+    selectSingleAsset: call.bind(null, 'Asset', false),
+    selectMultipleEntries: call.bind(null, 'Entry', true),
+    selectMultipleAssets: call.bind(null, 'Asset', true)
+  }
+
+  function call (entityType, multiple, options) {
+    options = Object.assign({}, options, {entityType, multiple})
```

Comment:
`Object.assign()` is [not supported by IE](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Browser_compatibility). SAD. I personally use the [`object-assign`](https://www.npmjs.com/package/object-assign) package.